### PR TITLE
Responsive Navbar 

### PR DESCRIPTION
**Pull request by meaghanfitzgerald**
This PR includes changes to fix the issues seen when resizing the navbar (see video attached). The Dropdown menu for language selection has been removed due to only our docs currently only supporting English. 

Before this PR: 
https://github.com/ava-labs/avalanche-docs/assets/99047250/fb2abce8-7988-480e-a99d-d810de055f96

After this PR:
https://github.com/ava-labs/avalanche-docs/assets/99047250/b8eebd8e-a7cd-4ab3-b690-3c61e85bb627

### CODE REVIEW

**Comment by Andyvargtz** [comment]
Not sure of what the issue might be, but I find the menu is not displaying correctly with some window size. 

https://github.com/ava-labs/avalanche-docs/assets/43045203/188a18c5-e5a5-49c6-be95-a7c04111203e

File: docusaurus.config.js [hunk 1]
```diff
@@ -135,10 +135,6 @@ const config = {
             sidebarId: "community",
             label: "Community",
           },
-          {
-            type: "localeDropdown",
-            position: "right",
-          },
```
Comment (connorch) [inline docusaurus.config.js:141]: Is this desired?  It removes the language dropdown:
https://meag-navbar-css.avalanche-docs.pages.dev/

Screenshot 2023-05-30 at 6 50 04 PM

Comment (meaghanfitzgerald) [reply docusaurus.config.js:141]: Yes, we currently only support English so the dropdown is misleading. Support for Spanish and Chinese will likely be added in the future.

**Comment by meaghanfitzgerald** [comment]
@Andyvargtz good catch, going to address this asap

**Comment by meaghanfitzgerald** [comment]
Closing this after a conversation with docusaurus team. The only current solution to this issue is to "Have fewer or shorter navbar items." At the moment the breakpoints aren't adjustable.

This aligns with our long terms goals to simplify the docs so best not to mess with the CSS too much to try to achieve it in the short term.

**Pull request closed without merging**